unpackaged/test/primary_affiliations/pre/classes/STG_UnmanagedInstall.cls

```diff
@@ -0,0 +1,13 @@
+public without sharing class STG_UnmanagedInstall {
```

Comment:
This is a much more elegant solution than anonymous apex since we are already deploying unmanaged metadata.